.github/workflows/check.yml

```diff
@@ -19,6 +19,21 @@ jobs:
       - name: Check Formatting
         run: deno fmt --check
 
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 'latest'
+
+      - name: Check Linting
+        working-directory: ./site
+        run: |
+          npm i -D markdownlint-cli2
```

Comment:
Why not `npm install`?

Reply:
Why not `npm ci`?

Reply:
With npm ci, it is not possible to install a specific dependency. So we can only install all dependencies, which is not optimal. However, the guarantee that npm ci will install only the version specified in the package.json is a good enough reason to use npm ci. If installing extra packages is not a problem, I will use npm ci.

Reply:
It isn't, go ahead. The important part is not to take more time than a build, but we're never gonna need that much time anyway

Reply:
That is, use npm ci?

Reply:
Yes.